### DISCUSSION TITLE
How to add accessible non-option child to Select?

### DISCUSSION THREAD

**Discussion by ghost** [adobe/react-spectrum#2910] [Q&A]
Hey, I am currently trying to create a select element with a custom element (see UnitSystemRadioGroup in image) inside.

I currently have implemented it using the useSelect hook and the component looks fine and the select functionality works fine.

### Question
How do I make the radio group accessible via keyboard?
I'd love to be able to let the user start (first focused element) at the first option and let him navigate using ArrowUp to the UnitSystemRadioGroup.
It seems to be already possible to use ArrowDown to go from the UnitSystemRadioGroup to the first option.
____

![Screenshot from 2022-03-03 10-38-03](https://user-images.githubusercontent.com/51439085/156537924-ce938d3e-ff6d-4658-bd3f-b0ceeb9cf0da.png)

### My current implementation
```typescript
// Return value of my Listbox component
<List {...listBoxProps} ref={ref}>
  <ListItem>
    <UnitSystemRadioGroup
      unitSystem={unitSystem}
      setUnitSystem={setUnitSystem}
    />
  </ListItem>
    {Array.from(state.collection).map((item) => (
      <Option key={item.key} item={item} state={state} />
    ))}
</List>
```

Everything else is pretty similar to the example in the `useSelect` documentation.

**Comment by snowystinger** [comment]
Hey, sorry for the delay, this fell off my radar. I will refer to you a comment about a similar kind of composite widget https://github.com/adobe/react-spectrum/issues/3020#issuecomment-1099434033
